dashboard/src/dictionaries/da.json

```
@@ -59,5 +59,159 @@
   "languages": {
     "da": "Dansk",
     "en": "Engelsk"
+  },
+  "components": {
+    "devices": {
+      "tables": {
+        "topOperatingSystems": "Top operativsystemer",
+        "mostCommonOperatingSystems": "Mest almindelige operativsystemer",
```

```suggestion
        "mostCommonOperatingSystems": "Mest anvendte operativsystemer",
```

Anvendte seems like a better translation to me 👍 If you agree you probably also agree that mostCommonBrowsers should be `Mest anvendte browsere`.